src/coreclr/jit/emitarm64.cpp

```diff
@@ -24419,11 +24408,19 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
         case IF_SVE_BV_2A_J: // ........xx..gggg ..hiiiiiiiiddddd -- SVE copy integer immediate (predicated)
             imm  = emitGetInsSC(id);
             code = emitInsCodeSve(ins, fmt);
-            code |= insEncodeReg_V_4_to_0(id->idReg1());                  // ddddd
-            code |= insEncodeReg_P_19_to_16(id->idReg2());                // gggg
+            code |= insEncodeReg_V_4_to_0(id->idReg1());   // ddddd
+            code |= insEncodeReg_P_19_to_16(id->idReg2()); // gggg
+
+            if (!isValidSimm8(imm))
+            {
+                assert(isValidSimm8_MultipleOf256(imm));
```

Comment:
we have done enough validation until we come here, so don't think the assert here adds any value. We do not usually assert in encoding portion of the code.